build/configs/rtl8730e/loadable_apps_8M/defconfig

```diff
@@ -824,12 +842,24 @@ CONFIG_LWIP_DHCPS_MAX_STATION_NUM=8
 # CONFIG_NETUTILS_FTPC is not set
 # CONFIG_NETUTILS_MDNS is not set
 # CONFIG_NETUTILS_FTPD is not set
+# CONFIG_NETUTILS_WEBSOCKET is not set
 # CONFIG_NETUTILS_LIBCOAP is not set
 # CONFIG_NETUTILS_TFTPC is not set
 # CONFIG_NETUTILS_TELNETD is not set
 # CONFIG_NETUTILS_SMTP is not set
 # CONFIG_NETUTILS_MQTT is not set
-# CONFIG_NET_SECURITY_TLS is not set
+CONFIG_NET_SECURITY_TLS=y
+CONFIG_TLS_WITH_HW_ACCEL=y
+CONFIG_TLS_MPI_MAX_SIZE=512
+# CONFIG_TLS_HAVE_NO_TIME_DATE is not set
+
+#
+# HW Options
+#
+CONFIG_TLS_HW_RNG=y
+CONFIG_TLS_HW_DH_PARAM=y
+CONFIG_TLS_HW_ECDH_PARAM=y
+CONFIG_TLS_HW_ECDSA_VERIFICATION=y
```

Comment:
Hello defconfig does not enable them. Is it ok to enable here?  Could you confirm this?